tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ng.html

```diff
@@ -28,7 +33,6 @@
         [header]="header"
         [sortingInfo]="sortingInfo"
         [hparamsEnabled]="hparamsEnabled"
-        disableContextMenu="true"
```

Comment:
I think this change should be behind a feature flag.

Reply:
SG, I'll put it behind the existing `enableScalarColumnCustomization` rather than creating a separate flag just for disabling context menus (context menus will be enabled when `enableScalarColumnCustomization` is enabled)

Reply:
Created another PR for the feature flag: #6742